pkg/state/labels.go

```diff
@@ -0,0 +1,5 @@
+package state
+
+const (
+	ForceRefreshLabel = "nmstate.io/force-nns-refresh"
```

Comment:
IMO this should live in the module of the feature. Having a module to catch all labels, no matter what is their purpose, is not much different from having utils.go. I'd move it to where it is used and make sure it is not exposed to underlying implementation - only to modules concerned about controllers

Reply:
Moved to `controllers/labels.go` since there are two controllers that use it.